packages/theme/etc/theme.api.md

```diff
@@ -729,6 +730,9 @@ export type TokenSetType = {
     [key: string]: TokenSetType | string | number | undefined;
 };
 
+// @public (undocumented)
+export type Variants = Record<string, any>;
```

Comment:
Variants should always contain base, something like:

```
interface Variants {
  base?: RecursivePartial<Tokens>;
  [key: string]: RecursivePartial<Tokens>;
} 
```

Reply:
I think this we should leave as is. I don't think base (changed to root) is required.